setup.py

```diff
@@ -43,7 +43,11 @@ def has_environment_marker_support():
 
 
 def main():
-    install_requires = ['py>=1.4.33', 'setuptools']  # pluggy is vendored in _pytest.vendored_packages
+    install_requires = [
+        'py>=1.4.33',
+        'setuptools',
+        'attrs',
```

Comment:
I might be missing something but didn't attrs just get vendored as well?  Why is it also unconditionally part of `install_requires`?